packages/gluestick/src/commands/test/test.js

```diff
@@ -36,6 +36,7 @@ const getJestDefaultConfig = (aliases: Object, webpackRules: RegExp[]): string[]
   }
 
   const config = {
+    testRegex: '\\/__tests__\\/.*\\.(test|spec)\\.jsx?$',
```

Comment:
Isn't this already the [default from Jest](https://facebook.github.io/jest/docs/configuration.html#testregex-string) ?